src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1219,9 +1217,24 @@ int fromCommandLine (
         nRet                    = rpcINTERNAL;
     }
 
-    std::cout << jvOutput.toStyledString ();
+    return { nRet, std::move(jvOutput) };
+}
+
+//------------------------------------------------------------------------------
+
+namespace RPCCall {
+
+int fromCommandLine (
+    Config const& config,
+    const std::vector<std::string>& vCmd,
+    Logs& logs)
+{
+    auto const result = rpcClient(vCmd, config, logs);
+
+    if (result.first != 1)
```

Comment:
The call chain of `rpcClient` isn't too bad, and `EXIT_FAILURE` and `EXIT_SUCCESS` are just `#define`s (at least in VS) anyway, so using the symbols will increase readability a bit without really any change to the binary.